universe/Planet.h

```diff
@@ -195,12 +195,39 @@ class FO_COMMON_API Planet final : public UniverseObject {
     template <typename T> friend void boost::python::detail::value_destroyer<false>::execute(T const volatile* p);
 
     static constexpr auto planet_meter_types = []() {
+#if defined(__cpp_using_enum)
         using enum MeterType;
+#else
+        static constexpr auto METER_POPULATION = MeterType::METER_POPULATION;
```

Comment:
Can we use boost.preprocessor loop here?

Reply:
could do it with standard macros, like most of the enums are defined with...

or could just put all the `MeterType::` back in the various uses below these `using enum` lines.

or could use compilers that actually support C++20 features.